source/site_ops/how-tos/add-waffle-flag-for-user.rst

```diff
@@ -0,0 +1,13 @@
+How to Enable a Waffle Flag for a User
+#######################################
+
+Follow these steps to enable a waffle flag for internal testing or limited releases:
```

Comment:
```suggestion
Follow these steps to enable a waffle flag for a single user. This may be useful in certain situations, such as internal testing or limited releases:
```